### PR TITLE
Add BeautifulSoup

### PR DESCRIPTION
**Pull request by Google-Autofuzz**
BeautifulSoup is the standard to parse untrusted/invalid/weird
html in Python, so fuzzing it for unexpected exceptions and
infinite loops makes sense. Moreover,
it's using various parsers, with some of them written in C.

### CODE REVIEW

File: projects/bs4/build.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+# Build fuzzers in $OUT.
+for fuzzer in $(find $SRC -name '*_fuzzer.py'); do
+  fuzzer_basename=$(basename -s .py $fuzzer)
+  fuzzer_package=${fuzzer_basename}.pkg
+  pyinstaller --distpath $OUT --onefile --name $fuzzer_package $fuzzer
+
+# Create execution wrapper.
+echo "#!/bin/sh
+# LLVMFuzzerTestOneInput for fuzzer detection.
+LD_PRELOAD=\$(dirname "\$0")/libclang_rt.asan-x86_64.so \
```
Comment (inferno-chromium) [inline projects/bs4/build.sh:28]: This wrapper has changed, please see latest docs https://google.github.io/oss-fuzz/getting-started/new-project-guide/python-lang/

File: projects/bs4/project.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+homepage: "https://www.crummy.com/software/BeautifulSoup/"
+main_repo: "https://code.launchpad.net/~leonardr/beautifulsoup/bs4"
+language: python
+primary_contact: "security-tps@google.com"
```
Comment (inferno-chromium) [inline projects/bs4/project.yaml:4]: need to remember to get dev contact .

**Review by inferno-chromium — CHANGES_REQUESTED** [review]
Some minor changes needed, see comments.

File: projects/bs4/project.yaml (hunk 2 above)
Comment (Google-Autofuzz) [reply projects/bs4/project.yaml:4]: Since I'll be receiving the alerts, I won't forget :)

**Review by inferno-chromium — APPROVED** [review]

**Pull request merged**